### PR TITLE
Fix URL to web, desktop settings pages on Desktop application 

### PR DESCRIPTION
**Pull request by HusainBhattiwala**
Fixed when user clicks on open khoj web settings for api token its been redirected to incorrect url hostUrl/config instead of hostUrl/settings @sabaimran @debanjum

### CODE REVIEW

File: src/interface/desktop/preload.js [hunk 1]
```diff
@@ -82,5 +82,5 @@ contextBridge.exposeInMainWorld('appInfoAPI', {
 
 contextBridge.exposeInMainWorld('navigateAPI', {
   navigateToSettings: () => ipcRenderer.send('navigate', 'config.html'),
```
Comment (debanjum) [inline src/interface/desktop/preload.js:84]: Should this be also updated to navigate to the desktop app's (new) settings page?

```suggestion
  navigateToSettings: () => ipcRenderer.send('navigate', 'settings.html'),
```

File: src/interface/desktop/chat.html [hunk 2]
```diff
@@ -475,7 +475,7 @@
             renderMessage(first_run_message, "khoj", null, null, true);
 
             // Open relevant setup links
-            document.getElementById("first-run-web-config").addEventListener('click', (event) => window.navigateAPI.navigateToWebSettings('config'));
+            document.getElementById("first-run-web-config").addEventListener('click', (event) => window.navigateAPI.navigateToWebSettings('settings'));
             document.getElementById("first-run-desktop-config").addEventListener('click', (event) => window.navigateAPI.navigateToSettings('config.html'));
```
Comment (debanjum) [inline src/interface/desktop/chat.html:479]: Should this be also updated to navigate to the desktop app's (new) settings page?

```suggestion
            document.getElementById("first-run-desktop-config").addEventListener('click', (event) => window.navigateAPI.navigateToSettings('settings.html'));
```

**Review by debanjum — COMMENTED** [review]
Thanks for creating a PR to fix this issue. I've left some comments

We may also need to update the desktop apps `main.js` file to use `webContent.getURL().includes('settings')` instead of `webContent.getURL().includes('config')`.

Let me know if you have any questions?

**Comment by HusainBhattiwala** [comment]
sure will do that

File: src/interface/desktop/preload.js (hunk 1 above)
Comment (HusainBhattiwala) [reply src/interface/desktop/preload.js:84]: yes i was going to ask that

File: src/interface/desktop/chat.html (hunk 2 above)
Comment (HusainBhattiwala) [reply src/interface/desktop/chat.html:479]: yes

**Comment by HusainBhattiwala** [comment]
have make the reviewed changes @debanjum @sabaimran

File: src/interface/desktop/main.js [hunk 3]
```diff
@@ -256,7 +256,7 @@ function pushDataToKhoj (regenerate = false) {
     .catch(error => {
         console.error(error);
         state["completed"] = false;
-        if (error?.response?.status === 429 && (BrowserWindow.getAllWindows().find(win => win.webContents.getURL().includes('config')))) {
+        if (error?.response?.status === 429 && (BrowserWindow.getAllWindows().find(win => win.webContents.getURL().includes('settings')))) {
             state["error"] = `Looks like you're out of space to sync your files. <a href="https://app.khoj.dev/settings#subscription">Upgrade your plan</a> to unlock more space.`;
             const win = BrowserWindow.getAllWindows().find(win => win.webContents.getURL().includes('config'));
```
Comment (debanjum) [inline src/interface/desktop/main.js:261]: This line may need to be updated too?

```suggestion
            const win = BrowserWindow.getAllWindows().find(win => win.webContents.getURL().includes('settings'));
```

**Review by debanjum — APPROVED** [review]
One other minor change maybe required. But otherwise changes look good to merge from my end!

File: src/interface/desktop/main.js (hunk 3 above)
Comment (HusainBhattiwala) [reply src/interface/desktop/main.js:261]: sorry i forget that can fix now??

Comment (debanjum) [reply src/interface/desktop/main.js:261]: That would be great!

Comment (HusainBhattiwala) [reply src/interface/desktop/main.js]: @debanjum i have added minor changes let me know

**Review by debanjum — APPROVED** [review]
Changes look good from my end. Thanks for the quick work! @sabaimran let me know if you want to have a look at the PR too? Else we can merge this

**Pull request merged**